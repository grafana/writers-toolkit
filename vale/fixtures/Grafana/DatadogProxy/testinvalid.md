Don't use:

- The Datadog proxy
- The datadog proxy
- Datadog Proxy
