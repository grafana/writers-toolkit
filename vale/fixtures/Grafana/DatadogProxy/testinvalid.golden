testinvalid.md:3:3:Grafana.DatadogProxy:Use 'Datadog proxy' instead of 'The Datadog proxy'.
testinvalid.md:4:3:Grafana.DatadogProxy:Use 'Datadog proxy' instead of 'The datadog proxy'.
testinvalid.md:5:3:Grafana.DatadogProxy:Use 'Datadog proxy' instead of 'Datadog Proxy'.
